packages/wrangler/src/type-generation/runtime/log-runtime-types-message.ts

```diff
@@ -0,0 +1,79 @@
+import { existsSync, readFileSync } from "fs";
+import { logger } from "../../logger";
+
+/**
+ * Constructs a comprehensive log message for the user after generating runtime types.
+ */
+export function logRuntimeTypesMessage(outFile: string, tsconfigPath: string) {
+	const existingTypes = readExistingTypes(tsconfigPath);
+	const isWorkersTypesInstalled = workersTypesEntryExists(existingTypes);
+	const updatedTypesString = buildUpdatedTypesString(existingTypes, outFile);
+
+	const message = isWorkersTypesInstalled
+		? `\n📣 Replace the existing "@cloudflare/workers-types" entry with the generated types path:`
```

Comment:
I'm not sure what is the convention for using emojis here. Without them it's a little dull and harder to read, but maybe we have another way to do this?

Reply:
Emojis are fun, so this seems fine to me? 🤷